src/screens/DiscoveryScreenV2/index.js

```diff
@@ -43,9 +44,14 @@ const DiscoveryScreenV2 = ({route}) => {
   const [searchText, setSearchText] = React.useState('');
   const [isFocus, setIsFocus] = React.useState(true);
   const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
+  const [tabs, setTabs] = React.useState({Users: 0, Communities: 0, Domains: 0, News: 0});
 
-  const [, discoveryDispatch] = React.useContext(Context).discovery;
+  const [discoveryData, discoveryDispatch] = React.useContext(Context).discovery;
+  const [profileState] = React.useContext(Context).profile;
   const cancelTokenRef = React.useRef(axios.CancelToken.source());
+  const [, followingDispatch] = React.useContext(Context).following;
+
+  const navigation = useNavigation();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New state variable `tabs` and context variables `profileState`, `followingDispatch` are introduced. Ensure that these new variables are used properly throughout the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->